components/ws-daemon/pkg/dispatch/dispatch.go

```diff
@@ -199,7 +199,8 @@ func (d *Dispatch) handlePodUpdate(oldPod, newPod *corev1.Pod) {
 			},
 		}
 
-		waitForPodCtx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
+		// Important!!!!: ideally this timeout must be equal to ws-manager https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager/pkg/manager/manager.go#L171
```

Comment:
Could use change this URL? Because it's persistent.

https://github.com/gitpod-io/gitpod/blob/2d898bf91a7962ad0e707ef82ff85c006626d2c1/components/ws-manager/pkg/manager/manager.go#L171

Reply:
This will be correct once we merge the change